extensions/switch-dns/src/config.ts

```diff
@@ -0,0 +1,9 @@
+export const DNS_LIST = [
+  { title: "Alibaba", dns: "223.5.5.5,223.6.6.6", accessory: "alidns.com" },
+  { title: "Google", dns: "8.8.8.8,8.8.4.4", accessory: "Google" },
+  { title: "Cloudflare", dns: "1.1.1.1,1.0.0.1", accessory: "Cloudflare DNS" },
+  { title: "114", dns: "114.114.114.114,114.114.115.115", accessory: "114 Public DNS" },
+  { title: "Empty DNS", dns: "Empty", accessory: "Empty DNS" },
```

Comment:
I'd probably call this "Revert to network default" – there are always DNS servers set, for most networks, but this is often provided by default by the network. It'll normally be the ISP.

A common use case I have for changing DNS is when I connect to a public WiFi network that doesn't support custom DNS, so I want to revert to the network default DNS.